core/model/src/main/java/org/activityinfo/model/query/DiscreteColumnView.java

```diff
@@ -0,0 +1,10 @@
+package org.activityinfo.model.query;
+
+public interface DiscreteColumnView extends ColumnView {
```

Comment:
Can you call this "EnumColumnView"? The DiscreteColumnViewXX are space-optimized implementations.